### DISCUSSION TITLE
How to specify a value in default file from command line

### DISCUSSION THREAD

**Discussion by laurent-martin** [jgm/pandoc#9487] [Q&A] [answered]
I am using the following default file:
https://github.com/IBM/aspera-cli/blob/main/docs/manual_pandoc_defaults.yaml

and pandoc is called from this makefile:
https://github.com/IBM/aspera-cli/blob/main/docs/Makefile#L31

and some pictures are referenced from this latex header:
https://github.com/IBM/aspera-cli/blob/main/docs/manual_include_in_header.tex

I use the standard template.

I have tried numerous ways to specify where to get **image** files from, using a variable on command line of pandoc, but never managed to make it work.

So, currently I just do not specify a path, just image file name, and images files are taken from CWD..

Any idea on how to specify the path as a variable from command line ?

**Comment by jgm** [comment]
did you try `--resource-path`?

**Comment by laurent-martin** [comment, marked as answer]
`--resource-path` did not work to specify images location.

Since I use LuaLatex as backend, finally, I did:

```tex
\graphicspath{{\directlua{tex.sprint(os.getenv('GFX_DIR'))}}}
```

and set env var GFX_DIR:

```bash
GFX_DIR=/path/to/folder pandoc ...
```
